src/myUtilities/system_utilities/SystemUtilities.java

```diff
@@ -1,4 +1,4 @@
-package myUtilities;
+package myUtilities.system_utilities;
```

Comment:
By convention in Java, package names are all lower case and without underscore (unless the name is hyphenated)

```suggestion
package myutilities.systemutilities;
```

Reply:
Also, I recommend dropping the "my" prefix in you packages and filenames.